src/protos/metadata.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package protos;
+
+// Metadata of a given file chunk
+message FileChunkMetadata {
+  // An immutable and globally unique 64 bit chunk identifier.
+  // This is assigned by the master at the time of chunk creation.
+  string chunk_handle = 1;
```

Comment:
nit: why a string and not a 64-bit int type?

Reply:
if it is a uint64_t it would match the atomic<uint64_t> that the master uses internally to generate these unique chunk handles?

Reply:
1. protocol buffer integer types is *different* from c++ integer types, so there’s no things such as uint64_t supported in protocol definitions
2. the design for master’s chunk handle is not finalized
3. even though paper uses 64bit, our design chooses UUID, which is what is used for globally unique identifiers, since using UUID strings as identifiers is the conventions nowadays and we can use libraries for that to ensure it’s unique. However, UUIDs are all strings. I can update the documentation to clarify this

Reply:
Also, a side note, for scalability in practice, especially in schema definition, always use strings for identifiers (design principles from google internal convention) as they are more extensible and easy for cross codebase work